doc/manual/introduction.md

```diff
@@ -74,18 +74,16 @@ website.
 
 ## Current state and roadmap
 
-Nickel is currently released in version `1.3`. We expect the core design of the
-language to be stable and the language to be useful for real-world applications.
-The next steps we plan to work on are:
+Nickel is currently released in version `1.4`. The core design of the language
```

Comment:
Did we forget updating that, again? Or did this change sneak in from somewhere elese?